src/components/Icon.js

```diff
@@ -58,8 +87,10 @@ Icon.propTypes = {
 }
 
 const StyledIcon = styled(Icon)`
-  fill: ${(props) => props.theme.colors.secondary};
-  &:hover path:last-of-type {
+  fill: ${(props) =>
+    props.color ? props.color : props.theme.colors.secondary};
+
+  &:hover path {
```

Comment:
I think this change is creating some weird behavior on the nav dropdown icons:

https://user-images.githubusercontent.com/8097623/145498288-95bbeb52-7c33-4454-99b9-2cfc7cb5e809.mp4

Reply:
Yeah, still seeing this as well

Reply:
Just pushed a patch for this